packages/react-components/src/components/styles/VideoTile.styles.ts

```diff
@@ -103,5 +103,6 @@ export const isSpeakingBorderDiv: IStyle = {
   height: '100%',
   position: 'absolute',
   top: '0',
-  left: '0'
+  left: '0',
+  pointerEvents: 'none'
```

Comment:
Why is this part of the PR?

Reply:
(from our debugging) Assumming this is what was capturing the click event?
If so make sure to add a comment why this css prop was added
```suggestion
  // Ensure the isSpeaking element does not steal any pointer events such as onClick events
  pointerEvents: 'none'
```